### PR TITLE
Changed deprecated expressions

### PR DESCRIPTION
**Pull request by bladeoner**
@wardviaene I have updated the deprecated expressions.

### CODE REVIEW

File: blob-storage/instance.tf [hunk 1]
```diff
@@ -3,7 +3,7 @@ resource "azurerm_virtual_machine" "demo-instance" {
   name                  = "${var.prefix}-vm"
   location              = var.location
   resource_group_name   = azurerm_resource_group.demo.name
-  network_interface_ids = ["${azurerm_network_interface.demo-instance.id}"]
+  network_interface_ids = ["azurerm_network_interface.demo-instance.id"]
```
Comment (wardviaene) [inline blob-storage/instance.tf]: here and in the next ones, the quotes are still there, so those should be removed

Comment (bladeoner) [reply blob-storage/instance.tf]: Thanks, I will update the PR

Comment (bladeoner) [reply blob-storage/instance.tf]: @wardviaene I have removed the quotes.

**Comment by bladeoner** [comment]
@wardviaene I have removed the quotes.

**Comment by wardviaene** [comment]
Thank you!

**Pull request merged**